src/routes/docs/tooling/command-center/keyboard-shortcuts/+page.markdoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: article
+title: Keyboard Shortcuts
+description: Learn to navigate the Appwrite Console efficiently and effectively with your keyboard
+---
+
+The Appwrite Console was designed with a keyboard first approach. Developer experience is greatly improved by virtue of the keyboard's ease of use, quick and seamless execution of essential actions and the consistency of shortcut keys to traverse the Appwrite Console.
+
+# Shortcuts {% #shortcuts %}
+
+The Appwrite Console supports keyboard shortcuts that make it easier to navigate and perform common actions quicker. The shortcuts use the following pattern: use the first letter from the call to action followed by the resource, product, service, or page you're targeting.  For example, the shortcut keys `G` + `S` navigates to the project's Storage screen. Similarly, the shortcut `G` + `F` navigates to the project's Functions screen. However, there are some exceptions. The shortcut `G` + `E` navigates to project Settings, because `G` + `S` is in use for Storage. This pattern is used to make usage of shortcuts consistent and predictable.
+
+## Global shortcuts {% #global-shortcuts %}
+
+Developers can also utilize global shortcuts anywhere in the Appwrite Console, allowing instant access to these features from any page on the Console.
+
+|PC (Windows / Linux)|macOS|Action|
+|----|----|----|
+|`Ctrl` + `K`|`⌘` + `K`|Access Command Center|
```

Comment:
Please move this down so it's the second table. Then, add a sub section for Command Center. Briefly explain that the command center shows the most relevant shortcuts based on the page you're on. With the Command Center, you can search for commands or content on the current page. Then, link to the command center page.

This snippet is helpful for SEO and visibility. Because it's a sub section and the other shortcuts are in the parent section, we need to move the content so the sub section comes after.